src/components/Stats/Stats.test.js

```diff
@@ -10,6 +10,9 @@ afterEach(() => {
 describe('<Stats />', () => {
   test('Component renders', async () => {
     const { container } = await render(<Stats />);
+    console.log(container.firstChild);
```

Comment:
is this console.log needed? just want to point out in case you missed it, but awesome job with the tests!